README.md

```diff
@@ -36,7 +36,7 @@ Fique à vontade para contribuir e melhorar a qualidade deste conteúdo, enviand
 
 ## Títulos
 
-* Leia online (grátis!): ["Iniciando"](up\%20&\%20going/README.md#you-dont-know-js-iniciando), Publicado: [Compre agora (inglês)](http://shop.oreilly.com/product/0636920039303.do), ebook format is grátis!
```

Comment:
No arquivo original ([em inglês](https://github.com/getify/You-Dont-Know-JS/tree/1st-ed#titles)) consta a expressão:
> but the ebook format is free!

Corrige essa tradução, por favor, mantendo a frase.